### DISCUSSION TITLE
Maya LT support?

### DISCUSSION THREAD

**Discussion by olegzvg** [Autodesk/maya-usd#2099] [Q&A] [answered]
Hello, is there a way to install Maya USD to Maya LT? 

I tried it with the trial version, but USD didn't show up. 
I'm looking for a package which able to rig and animate USD directly. Seemingly, besides Presto, Maya is the closest thing to that. So is there a way to use LT for that?

**Comment by kxl-adsk** [comment, marked as answer]
Not something I tried explicitly, but I don't expect MayaUSD to load in MayaLT given the third party and python limitations it has. Not to mention that Maya and Maya LT are ABI incompatible, so you would need new installers that are just for LT.

**Reply by olegzvg** [reply]
pretty sad

**Reply by olegzvg** [reply]
@kxl-adsk Should it work with Maya Indie, what do you think?

**Reply by kxl-adsk** [reply]
I don't know a technical reason why it wouldn't...but don't make any purchasing decision based on that. I'm hoping someone with an Indie license can chime in here to confirm.

**Reply by stkrake** [reply]
Yes, Indie works. Indie seems to be the exact same binary (if you start with the 30-day trial and then buy Indie, nothing has to be updated).

**Reply by olegzvg** [reply]
perfect, thank you very much
